pkg/cvo/availableupdates.go

```diff
@@ -71,6 +71,17 @@ func (optr *Operator) syncAvailableUpdates(ctx context.Context, config *configv1
 	current, updates, conditionalUpdates, condition := calculateAvailableUpdatesStatus(ctx, string(config.Spec.ClusterID),
 		transport, userAgent, upstream, desiredArch, currentArch, channel, optr.release.Version, optr.conditionRegistry)
 
+	if au := optr.getAvailableUpdates(); au != nil {
```

Comment:
Nit: Could we use the already defined variable `u` from the line 48?
```suggestion
	if u != nil {
```